tests/math.py

```diff
@@ -7,27 +7,26 @@
 #   + this disables some rules and tests
 # * the last three tests
 
-from snake_egg import EGraph, Rewrite, Var, vars
```

Comment:
I realized these tests got formatted with [black](https://github.com/psf/black)  and [isort](https://github.com/pycqa/isort). If you prefer a more minimal diff, I can revert them back, and just keep the needed changes (added the `#type: ignore`s and renamed the `namedtuples`s)

Reply:
Yes, I think a more minimal diff would be good for now!

Reply:
Sonds good, I reverted them back to the originals and only added the minimal I could to make it pass with MyPy.